### ISSUE TITLE
friendsofphp/php-cs-fixer version 3.59.3 broke fixer

### ISSUE THREAD

**Issue by cobyl** [PedroTroller/PhpCSFixer-Custom-Fixers#209]
Since  friendsofphp/php-cs-fixer version 3.59.3 fixer return this error:

```
PHP Fatal error:  Class PedroTroller\CS\Fixer\Behat\OrderBehatStepsFixer contains 1
abstract method and must therefore be declared abstract or implement the remaining methods
(PhpCsFixer\Fixer\ConfigurableFixerInterface::configure) in vendor/pedrotroller/php-cs-custom-fixer/src/PedroTroller/CS/Fixer/Behat/OrderBehatStepsFixer.php on line 16

Fatal error: Class PedroTroller\CS\Fixer\Behat\OrderBehatStepsFixer contains 1 abstract method
 and must therefore be declared abstract or implement the remaining methods (PhpCsFixer\Fixer\ConfigurableFixerInterface::configure) in
 vendor/pedrotroller/php-cs-custom-fixer/src/PedroTroller/CS/Fixer/Behat/OrderBehatStepsFixer.php on line 16
```

**Comment by jorismak** [comment]
https://github.com/PedroTroller/PhpCSFixer-Custom-Fixers/pull/210

**Comment by PedroTroller** [comment]
Fixed with [v2.33.1](https://github.com/PedroTroller/PhpCSFixer-Custom-Fixers/releases/tag/v2.33.1), thanks @jorismak !!!
